source/versoSpoof.js

```diff
@@ -116,3 +117,11 @@ module.exports.propertyTypes = [
     'json': ['literal', 'resource', 'lookup', 'target', 'list']
   }
 ]
+
+module.exports.valuesFromConfig = [
```

Comment:
picky, non-blocking feedback (she's baaaaaAAAAaaaaack):

is this super-temporary like this?  Because as with a previous PR - this is nothing to do with verso.  I can show you how to add files to the github pages if you want to have this file live at http://ld4p.github.io/sinopia or maybe there could be a comment in here for "this here until this info is stored in sinopia server" with a ticket number, or ...

I'm sure this is a temporary thing, but again ... it's confusing that it's under things named "verso" just b/c you found that expeditious.   The data itself could even be in '../sample_lookupConfig.json' instead of in the sample_data_from_verso directory ...

it's a small thing, but I would never think to look for this code here!

Reply:
Ticket #168 is for making this work with sinopia server, but we will still need this spoof for the integration tests, so I prefer to have it spoofed locally.